src/components/FormSubmit/index.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import lodashGet from 'lodash/get';
+import {View} from 'react-native';
+import {propTypes, defaultProps} from './formSubmitPropTypes';
```

Comment:
```suggestion
import * as formSubmitPropTypes from './formSubmitPropTypes';
```

Please follow this syntax.

Reply:
Let's just update the files whichever changed by you